front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -147,173 +131,131 @@ const TimetableTrainCard = ({
-          onClick={() => changeSelectedTrainId(train.id)}
-        >
-          <div className="scenario-timetable-train-header">
-            <div className="scenario-timetable-train-name">
-              <div
-                className={cx('scenario-timetable-train-idx', {
-                  projected: projectionPathIsUsed,
-                })}
-              >
-                {idx + 1}
+        <div className="base-info">
+          <div className="title-img">
+            <div className="checkbox-title">
+              {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+              <div onClick={(e) => e.stopPropagation()}>
+                <Checkbox
+                  label=""
+                  className="mr-2"
+                  checked={isInSelection}
+                  onChange={() => handleSelectTrain(train.id)}
+                  small
+                />
               </div>
-              {train.invalidReason && (
-                <div
-                  className="mr-1 scenario-timetable-train-invalid-icons"
-                  title={train.invalidReason}
-                >
-                  {getInvalidIcon(train.invalidReason)}
-                </div>
-              )}
-              <div className="train-name" title={train.trainName}>
+              <div title={train.trainName} className="checkbox-label">
                 {train.trainName}
               </div>
-              {train.rollingStock && (
-                <span className="img-container">
-                  <RollingStock2Img rollingStock={train.rollingStock} />
-                </span>
-              )}
             </div>
-            <div
-              className={cx('scenario-timetable-train-times', {
-                'not-honored': train.notHonoredReason === 'scheduleNotHonored',
-                'too-fast': train.notHonoredReason === 'trainTooFast',
-              })}
-            >
-              {train.notHonoredReason === 'scheduleNotHonored' && (
-                <div className="ml-1">
-                  <Clock size="lg" />
-                </div>
-              )}
-              {train.notHonoredReason === 'trainTooFast' && (
-                <div className="ml-1">
-                  <Flame size="lg" />
+            <div className="rolling-stock">
+              {train.rollingStock && !train.invalidReason && (
+                <div className="rolling-stock-img">
+                  <RollingStock2Img rollingStock={train.rollingStock} />
                 </div>
               )}
-              <div>
-                <div className="scenario-timetable-train-departure">
-                  {formatDateTimeWithoutYear(train.startTime)}
-                </div>
-                <div className="scenario-timetable-train-arrival" title={train.arrivalTime}>
-                  {formatDateTimeWithoutYear(train.arrivalTime)}
-                </div>
+              {train.invalidReason && t(`timetable.invalid.${train.invalidReason}`)}
```

Comment:
Seems like on large screens, the position of the invalid reason is too much on left

![image](https://github.com/user-attachments/assets/c65615bd-110c-4b96-9c50-7ee36311eb15)

Reply:
Done
@thibautsailly sent me this screen
mockup: ![image](https://github.com/user-attachments/assets/745f4769-eb19-4ed3-aadc-eb786ad63096)
app: 
<img width="420" alt="image" src="https://github.com/user-attachments/assets/3966b46b-2151-432a-8d3f-e6e74992cd1a">